CHANGELOG.md

```diff
@@ -0,0 +1,110 @@
+# Changelog
+
+
+
+## [Unreleased]
```

Comment:
The idea of `[Unreleased]` is that it represents a placeholder for the next version number. At the time of creating the PR, it might not be clear which exact version the feature of the PR  will belong to. Thus my thought was to just generalise it and define the exact release version when releasing the new version.